RELEASE.md

```diff
@@ -14,26 +14,39 @@
     * Limitador version the operator enables installations of (without prefix) – i.e. `0.X.Y`
     * If the release is a prerelease
 
-3. Verify that the build [release tag workflow](https://github.com/Kuadrant/dns-operator/actions/workflows/build-images-for-tag-release.yaml) is triggered and completes for the new tag
+3. Verify that the build [release tag workflow](https://github.com/Kuadrant/limitador-operator/actions/workflows/build-images-for-tag-release.yaml) is triggered and completes for the new tag
 
 4. Verify the new version can be installed from the catalog image, see [Verify OLM Deployment](#verify-olm-deployment)
```

Comment:
```suggestion
4. Verify the new version can be installed from the catalog image
```
I think we can remove the link given it's the very next section. wdyt?

Reply:
ok. I will do the change myself as if I commit your suggestion, DCO check complains

Reply:
done

Reply:
oh yeah i never use the commit in github. partly for DCO but also for nice commit history. I thought we don't tbh. Only to show a suggestion.